src/layout/layout.js

```diff
@@ -402,6 +408,15 @@
     }
   };
 
+  /**
+   * Constructor for an individual tab.
+   *
+   * @constructor
+   * @param {HTMLElement} tab The HTML element for the tab.
+   * @param {Array.<HTMLElement>} tabs Array with HTML elements for all tabs.
```

Comment:
Not a big deal, but the '.' between Array and <HTMLElement> is now no longer required.